fe/src/features/projects/CreateProject.vue

```diff
@@ -21,6 +21,7 @@
                 to="/projects"
                 glow-color="#00F1AF"
                 shadow-strength=2
+                style="color: red !important; font-weight: bold"
```

Comment:
I don't really like the design of this button. Is it possible to maybe make the button white with green outline and black text? I tried but adding `outlined` does not add an outline, so I don't know if this is because of some class or something?

Reply:
yeah I was just trying something to fix the White on Green since it's not readable. I will undo this for now